### PR TITLE
Batch submission revamp (slurm + post-process + ...)

### PR DESCRIPTION
**Pull request by jcblemai**
Blockers:
- [x] merge slurm_batch and aws submission script; limit duplicated code
- [x] streamline submission process to be hardware agnostic
- [x] Conda enviroment that works end-to-end
- [x] test that slurms gives the same results as AWS
- [x] resume from s3 on slurm tested
- [x] Slurm upload to s3 after run tested
- [x] post-process job dependency on slurm tested
 - [x] send phone notification on run done
- [x] slurm submission work 
- [x] Full post-processing from slurm run
- [x] aws: check if works with the new script
- [x] Write doc on slurm submission
- [x] fix discard seeding bug
- [x] harmonize batch enviroment variable to prevent bad errors
- [x] clean log and highlight error

Nice to have:
- [x] Slack integration
- [x] Remove annoying warning from pandas that pollute logs
- [ ] doc: introduce slurm and file system
- [x] add cdlTools for COVID runs
- [x] Check with HPC manager if our use is ok ? Concerned about N files per Tb
- [x] Put some helpers (around filesystem) in bash_rc
- [x] prep script for pipeline installation
- [ ] Test resume from filesystem files
- [ ] aws: check post processing
- [x] Allow only one geoid in gempyor
- [ ] error management on SLURM
- [ ] remove folder list on AWS batch
- [ ] harmonize slurm log
- [ ] Check what is saved to s3

Nicer to have: (Check rest todo in case we can ship more in this PR):
- [ ] make so we can skip writing seir file, except the first and last.
- [ ] magic numbers
- [ ] exceptions for each stage as they are fast
- [ ] gempyor to pip

### CODE REVIEW

File: batch/AWS_inference_runner.sh [hunk 1]
```diff
@@ -66,29 +68,30 @@ fi
 
 python -m pip install --upgrade pip # needs new pip for toml file
 
-(cd COVIDScenarioPipeline && pip install -e gempyor_pkg)
+(cd $COVID_PATH && pip install -e gempyor_pkg)
```
Comment (jcblemai) [inline batch/AWS_inference_runner.sh:71]: Does $COVID_PATH makes sense here as the tar.gz file should send the same folder

Comment (jcblemai) [reply batch/AWS_inference_runner.sh:71]: (2 occurences)

**Review by jcblemai — COMMENTED** [review]
ok till now

- [x] send phone notification
- [x] slurm submission work and tested
- [x] resume from s3 tested
- [ ] resume from fs tested
- [x] send to s3 tested
- [x] post-process dependency on slurm tested
- [ ] aws: check if works
- [ ] aws: check post processing

**Review by kjsato — DISMISSED** [review]
batch/inference_job.py
 in l. 188(+), comma is needed

**Comment by jcblemai** [comment]
@kjsato 

> batch/inference_job.py
> in l. 188(+), comma is needed


where is the comma needed on l. 188 ?

Screenshot 2023-02-14 at 15 41 29

(you can also comment on a specific code line with github in case)

**Review by saraloo — APPROVED** [review]

**Review by shauntruelove — APPROVED** [review]
Looks good

**Pull request merged**